ui/blocks/BlocksTableItem.tsx

```diff
@@ -119,7 +119,7 @@ const BlocksTableItem = ({ data, isLoading, enableTimeIncrement }: Props) => {
               { burntFees.dividedBy(WEI).toFixed(8) }
             </Skeleton>
           </Flex>
-          <Tooltip label={ isLoading ? undefined : 'Burnt fees / Txn fees * 100%' }>
+          <Tooltip label={ isLoading ? undefined : 'Reward pool fees / Txn fees * 100%' }>
```

Comment:
The same tooltip copy can be used here too `ZEN awarded to the Reward Pool from transactions included in the block (Base fee (per unit of gas) * Gas Used).`